src/routes/comment.routes.js

```diff
@@ -0,0 +1,118 @@
+
+
+const router = express.Router();
+
+/**
+ * @swagger
+ * tags:
+ *   name: comment
+ *   description: Trips activities
+ */
+
+/**
+ * @swagger
+ *
+ * /trip-request/comment/{tripId}:
+ *    post:
+ *      summary: user and manager can comment to the trip request
+ *      tags: [comment]
+ *      parameters:
+ *       - name: token
+ *         in: header
+ *         description: Check token authentication
+ *         required: true
+ *         type: string
+ *       - name: tripId
+ *         in: path
+ *         description: trip request ID
+ *         required: true
+ *         type: string
+ *      requestBody:
+ *        required: true
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/schemas/tripRequestcomment'
+ *      responses:
+ *        "201":
+ *          description: trip request comment schema
+ *
+ * components:
+ *    schemas:
+ *      tripRequestcomment:
+ *        type: object
+ *        required:
+ *          - comment
+ *        properties:
+ *          comment:
+ *            type: string
+ */
+router.post('/comment/:tripId', verifyToken.headerToken, Validate.tripRequestCommentValidation(), isValid, commentMiddleware.validateTipIdAndUserId, comment.createTripRequestComment);
+
+/**
+ * @swagger
+ * tags:
+ *   name: comment
+ *   description: Trips activities
+ */
+
+/**
+ * @swagger
+ *
+ * /trip-request/comment/{tripId}/view?page:
+ *    get:
+ *      summary: user and manager can view the trip request comment of
+ *      tags: [comment]
+ *      parameters:
+ *       - name: token
+ *         in: header
+ *         description: Check token authentication
+ *         required: true
+ *         type: string
+ *       - name: tripId
+ *         in: path
+ *         description: trip request ID
+ *         required: true
+ *         type: string
+ *       - name: page
+ *         in: query
+ *         schema:
+ *           type: integer
+ *         description: limit number
+ *         required: true
+ *      responses:
+ *        "200":
+ *          description: trip request comment schema
+ */
+router.get('/comment/:tripId/view', verifyToken.headerToken, commentMiddleware.validateTipIdAndUserId, comment.viewTripRequestComment);
+
+/**
+ * @swagger
+ *
+ * /trip-request/comment/{commentId}/delete:
+ *    delete:
+ *      summary: user and manager can delete the trip request comment
+ *      tags: [comment]
+ *      parameters:
+ *       - name: token
+ *         in: header
+ *         description: Check token authentication
+ *         required: true
+ *         type: string
+ *       - name: commentId
+ *         in: path
+ *         description: comment ID
+ *         required: true
+ *         type: string
+ *      responses:
+ *        "200":
+ *          description: trip request comment schema
+ */
+router.delete('/comment/:commentId/delete', verifyToken.headerToken, commentMiddleware.deleteCommentValidation, comment.deleteTripRequestComment);
```

Comment:
If you think about it, we do not need to include a verb in our routes if there's an HTTP method for that. The correct route should be " DELETE /trip-requests/:tripRequestID/comments/:commentID".